.github/workflows/ci.yml

```diff
@@ -0,0 +1,29 @@
+name: CI pipeline
+
+on:
+  push:
+    branches: [ "master" ]
+  #pull_request:
+    #branches: [ "master" ]
+
+jobs:
+
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v3
+    - name : Login Docker Hub
+      env:
+         DOCKER_USERNAME: ${{ secrets.DOCKER_USERNAME }}
+         DOCKER_PASSWORD: ${{ secrets.DOCKER_PASSWORD }}
+      run: docker login -u $DOCKER_USERNAME -p $DOCKER_PASSWORD
+      
+    - name: Build the Docker image
+      run: docker build -t hydroapp .
+    - name: Change the tag
+      run: docker tag hydroapp:latest nihaparasa/hydroapp:latest
```

Comment:
Can I get a url to a docker image already public on your DockerHub account? I; like to try and deploy from it one my own AWS instance.